src/gui/mzroll/mzfileio.cpp

```diff
@@ -1057,15 +1057,15 @@ void mzFileIO::_readPeakTablesFromSQLiteProject(const vector<mzSample*> newSampl
             group->setAdduct(DB.findAdductByName(group->getAdduct()->getName()));
 
         // assign group to bookmark table if none exists
-        if (group->searchTableName.empty())
-            group->searchTableName = "Bookmark Table";
+        if (group->tableName().empty())
+            group->setTableName("Bookmark Table");
```

Comment:
Make sure that the table name is updated when the bookmarked group is merged to any peak table.

Reply:
Will fix.